crm_compassion/models/event_compassion.py

```diff
@@ -138,7 +139,8 @@ class event_compassion(models.Model):
         track_visibility='onchange',
         required=True)
     planned_sponsorships = fields.Integer(
-        'Expected sponsorships', track_visibility='onchange', required=True)
+        'Expected sponsorships', compute='_compute_planned_sponsorships',

```

Comment:
No, this field must not be computed. It's the user who must estimate how many sponsorships we will win from this event.

The computation you made must be for the demand planning when we compute the demand for the events.